### ISSUE TITLE
How to change the order in date_histogram? and how to limit the document count in it?

### ISSUE THREAD

**Issue by milome** [NLPchina/elasticsearch-sql#536]
I tried the sql script below on the demo website:
select insert_time from online  group by date_histogram(field='insert_time','interval'='1.5h','format'='yyyy-MM','min_doc_count'=5)

But it fails with error： 
Error:{"message":"Error. Unable to parse ODBC Literal Timestamp","ok":false}

Meanwhile the default order of date_histogram is "asc", how to change it to "desc"?
Now I am using  version "5.1.2.0".
Thanks a lot!!

**Comment by shi-yuan** [comment]
```sql
select insert_time from online group by date_histogram(field='insert_time','interval'='1.5h','format'='yyyy-MM','min_doc_count'=5,'order'='desc')
```
```json
{
  "from": 0,
  "size": 0,
  "_source": {
    "includes": [
      "insert_time"
    ],
    "excludes": []
  },
  "stored_fields": "insert_time",
  "aggregations": {
    "date_histogram(field=insert_time,interval=1.5h,format=yyyy-MM,min_doc_count=5,order=desc)": {
      "date_histogram": {
        "field": "insert_time",
        "format": "yyyy-MM",
        "interval": "1.5h",
        "offset": 0,
        "order": {
          "_key": "desc"
        },
        "keyed": false,
        "min_doc_count": 5
      }
    }
  }
}
```

**Comment by milome** [comment]
Hi Shi-yuan,
Many thanks for your quick fix.

I tried to run it in the demo website:
http://www.nlpcn.org:9999/web/
with the sql:
select insert_time from online group by date_histogram(field='insert_time','interval'='1.5h','format'='yyyy-MM','min_doc_count'=5,'order'='desc')

But still, there is following error:
 Error:{"message":"date range err or not define field min_doc_count=5","ok":false}
Seems you just fixed the branch 5.1.2.1, while this demo website use a different version?
Thanks a lot.

**Comment by shi-yuan** [comment]
Yeah! [The demo website](http://www.nlpcn.org:9999/web/) use a different version!
